src/scenic/domains/driving/actions.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, steer: float):
 	def applyTo(self, obj, sim):
 		obj.setSteering(self.steer)
 
-class SetBrakeAction(SteeringAction):
+class SetBrakeAction(Action):
```

Comment:
Why make this change? This action is only supported by objects which support the steering protocol.

Reply:
If I remember correctly, I used to get a runtime error from Scenic's type-compatibility checker. Perhaps it's not an issue in the latest version of Scenic.

Reply:
OK, will revert this then, but let me know if you encounter any problems later.